packages/contracts/src/contracts/current/test/DummyERC721Token/DummyERC721Token.sol

```diff
@@ -56,4 +56,20 @@ contract DummyERC721Token is
         );
         _mint(to, tokenId);
     }
+
+    /**
+    * @dev Function to burn a token
+    * @dev Reverts if the given token ID doesn't exist
+    * @param tokenId uint256 ID of the token to be minted by the msg.sender
+    */
+    function burn(address owner, uint256 tokenId)
+        public
+        onlyOwner
+    {
+        require(
+            exists(tokenId),
+            "Token with tokenId does not exist."
+        );
+        _burn(owner, tokenId);
+    }
```

Comment:
Hm... I guess you still need to merge `v2-prototype` into this base branch of yours.